pallets/trustless-bridge/ethereum-light-client/src/lib.rs

```diff
@@ -1032,6 +1036,17 @@ pub mod pallet {
         })
     }
 
+    impl<T: Config> EthereumGasPriceOracle for Pallet<T> {
+        fn get_base_fee(
+            network_id: EVMChainId,
+            header_hash: H256,
```

Comment:
Why not return base fee for the best known header?

Reply:
Yes, I added to the new fees implementation, will add to this PR.